LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 Dr. Shahin Rostami
```

Comment:
Why did you add this license here?

Reply:
Thank you for catching this @ocefpaf. I used the GitHub front-end to generate the license and it places it in the repo root, regardless of the current dir. I'll delete it now.